src/main/kotlin/dk/cachet/carp/webservices/study/repository/CoreStudyRepository.kt

```diff
@@ -99,6 +99,7 @@ class CoreStudyRepository
                     studyStatus.studyId,
                     studyStatus.name,
                     studyStatus.createdOn,
+                    UUID(it.createdBy.toString()),
```

Comment:
if you call toString on a nullable property it is just going to be `"null"`, which is going to error out when you try to instantiate a `UUID`

Reply:
Instead, try to check if it exists in the first place and handle the request accordingly

Reply:
applying UUID(createdBy) auto checks if it exists in the first place if it is not the correct string format for UUID or its null; good enough;